packages/browser/jest.setup.js

```diff
@@ -2,7 +2,13 @@
 // https://github.com/jsdom/jsdom/issues/1612
 /* eslint-disable @silverhand/fp/no-mutation */
 /* eslint-disable unicorn/prefer-module */
+const { Crypto } = require('@peculiar/webcrypto');
```

Comment:
Will one of the following work:
1. Set jest env browser
2. See attached

![image](https://user-images.githubusercontent.com/14722250/153539082-50be3405-25fe-410c-95d4-fc910908a235.jpeg)

Reply:
@gao-sun

1. jest-environment `jsdom` does not work.
   See https://github.com/jsdom/jsdom/issues/1612 jsdom open issue: window.crypto is undefined
2. It's one of the existing solution. I'm good with both.

<img width="808" alt="image" src="https://user-images.githubusercontent.com/10594507/153543791-038d19d0-042f-4123-a93b-430aa36e0d80.png">

cc: @wangsijie

Reply:
Improved.
```ts
const crypto = require('crypto');

……

global.crypto = {
  getRandomValues: (buffer) => crypto.randomFillSync(buffer),
  subtle: crypto.webcrypto.subtle,
};
```